Makefile

```diff
@@ -19,7 +19,7 @@ OPEN_PR="true"
 # Force the installation of a Concrete Python version, which is very useful with nightly versions
 # /!\ WARNING /!\: This version should NEVER be a wildcard as it might create some
 # issues when trying to run it in the future.
-CONCRETE_PYTHON_VERSION="concrete-python==2.7.0"
+CONCRETE_PYTHON_VERSION="concrete-python==2.7.0.dev20240709"
```

Comment:
are we upgrading CP in this PR ? if so, please do it in the `pyproject.toml` as well. This makefile version is not relevant anymore now Concrete has a public repo, we just haven't removed it yet (cc https://github.com/zama-ai/concrete-ml-internal/issues/4463)

Reply:
I didn't know that. Yes upgrading since I can't use CML on intel for now